<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorControlView</class>
 <widget class="QWidget" name="CameraControlView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 44, 51);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255ï¼Œ255);</string>
         </property>
         <property name="text">
          <string>D2C Render</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SwitchButton" name="switch_d2c_render" native="true">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cb_resolution_rgb">
       <property name="styleSheet">
        <string notr="true">QComboBox {
	color: rgb(255, 255, 255);
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
 
 
QComboBox:!editable, QComboBox::drop-down:editable {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
 
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
 
QComboBox::down-arrow {
    image: url(:/res/image/combobox_ico.png);
}
 
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView
{
     background-color: rgba(62,77,89);
}
 
QComboBox QAbstractItemView::item
{
    color: rgb(255, 255, 255);
    height: 24px;
}
 
QComboBox QAbstractItemView::item:selected
{	
    background-color: rgba(9,11,13);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>D2CDistance</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="le_d2c_distance_value">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);</string>
       </property>
       <property name="text">
        <string>2000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_d2c_distance_set">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(193, 211, 227);
background-color: rgb(62, 77, 89);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/res/image/write.png</normaloff>:/res/image/write.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="slider_d2c_distance">
     <property name="maximum">
      <number>7000</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">src\weight\switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
